openedx/features/learner_profile/templates/learner_profile/learner_profile.html

```diff
@@ -4,28 +4,63 @@
 <%inherit file="/main.html" />
 <%def name="online_help_token()"><% return "profile" %></%def>
 <%namespace name='static' file='/static_content.html'/>
+
 <%!
 import json
 from django.core.urlresolvers import reverse
 from django.utils.translation import ugettext as _
 from openedx.core.djangolib.js_utils import dump_js_escaped_json
+from openedx.core.djangolib.markup import HTML, Text
 %>
 
 <%block name="pagetitle">${_("Learner Profile")}</%block>
 
 <%block name="bodyclass">view-profile</%block>
 
+<%block name="headextra">
+<%static:css group='style-course'/>
+</%block>
+
 <div class="message-banner" aria-live="polite"></div>
 <main id="main" aria-label="Content" tabindex="-1">
     <div class="wrapper-profile">
-        <div class="ui-loading-indicator">
-            <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
+        <div class="profile ${'profile-self' if own_profile else 'profile-other'}">
+            <div class="wrapper-profile-field-account-privacy"></div>
+            <div class="wrapper-profile-sections account-settings-container">
+                % if own_profile:
+                    <div class="profile-header">
+                        <div class="header">${_("My Profile")}</div>
+                        <div class="subheader">
+                            ${Text(_('Build out your profile to personalize your identity on {platform_name}.')).format(
+                                platform_name=platform_name,
+                            )}
+                        </div>
+                    </div>
+                % endif
+                <div class="ui-loading-indicator">
+                    <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
```

Comment:
This one is translated because it is in a production template, not a fixture. Anyway, it is a good thing to notice and report on missing translations.